examples/simple/MODULE.bazel

```diff
@@ -0,0 +1,39 @@
+bazel_dep(
+    name = "swift_gazelle_plugin",
+    version = "0.0.0",
+)
+local_path_override(
+    module_name = "swift_gazelle_plugin",
+    path = "../..",
+)
+
+bazel_dep(name = "cgrindel_bazel_starlib", version = "0.21.0")
+
+# The apple_support bazel_dep must come before the rules_cc.
+# https://github.com/bazelbuild/apple_support#incompatible-toolchain-resolution
+bazel_dep(name = "apple_support", version = "1.17.1")
+bazel_dep(
+    name = "rules_swift",
+    version = "2.2.2",
+    repo_name = "build_bazel_rules_swift",
+)
+
+apple_cc_configure = use_extension(
+    "@apple_support//crosstool:setup.bzl",
+    "apple_cc_configure_extension",
+)
+use_repo(apple_cc_configure, "local_config_apple_cc")
+
+# Dev Dependencies
+
+bazel_dep(
+    name = "bazel_skylib_gazelle_plugin",
+    version = "1.7.1",
+    dev_dependency = True,
+)
```

Comment:
Is all of this necessary in the example? I would think it just needs rules_swift, gazelle, and the swift_gazelle_plugin

Reply:
It thought so too. I tried removing `apple_support`, but ran into errors. I did not spend any time investigating.